test_image.sh

```diff
@@ -167,6 +168,29 @@ function stop_container {
 	docker rm "${CONTAINER_ID}" > /dev/null
 }
 
+function test_docker_image_bundle {
+	export CONTAINER_EXPORTED_PORT=$(docker port "${CONTAINER_ID}" 8080 | head -n 1 | awk -F: '{print $2}')
+
+	if [ ! -d "playwright-dependencies" ]
+	then
+		echo "Directory playwright-dependencies does not exist"
+
+		exit 1
+	fi
+
+	cd playwright-dependencies || exit
```

Comment:
@lucasmiranda0, is the `|| exit` really necessary since we're already checking the existence of the directory `playwright-dependencies`?